docs/docs/workers.md

```
@@ -233,10 +246,10 @@ queue = Queue('foo', serializer=JSONSerializer)
 worker = Worker([queue], serializer=JSONSerializer)
 ```
 
-When starting workers from the command line, pass the serializer path:
+When starting workers from the command line, pass the serializer (the shorthand `json` resolves to `rq.serializers.JSONSerializer`; a dotted import path also works):
 
 ```console
-$ rq worker --serializer rq.serializers.JSONSerializer
+$ rq worker --serializer json
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)